openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.cpp

```diff
@@ -385,6 +385,14 @@ ComponentType componentType(const HVACComponent & component)
     {
       return ComponentType::COOLING;
     }
+    case openstudio::IddObjectType::OS_Generator_MicroTurbine_HeatRecovery :
+    {
+      // Maybe that should be both, in the case of an absorption chiller?
```

Comment:
Well we probably aren't going to have a one size fits all solution. Use your best judgment about what you think the most common control scheme is and go with that. 

For custom stuff that is what the operation scheme API is for.